e2e/images_test.go

```diff
@@ -154,6 +141,8 @@ a-simple-app:latest simple
 			ExitCode: 1,
 			Err:      `could not tag 'a-simple-app:not-a-tag': no such application image`,
 		})
+		cmd.Command = dockerCli.Command("app", "image", "tag", "a-simple-app", "b@simple-app")
+		icmd.RunCmd(cmd).Assert(t, icmd.Expected{ExitCode: 1})
```

Comment:
Must be removed

Reply:
oh yes. I'm really bad at merge conflict resolution :face_with_head_bandage: